doc/api/vm.md

```diff
@@ -1573,6 +1573,90 @@ inside a `vm.Context`, functions passed to them will be added to global queues,
 which are shared by all contexts. Therefore, callbacks passed to those functions
 are not controllable through the timeout either.
 
+## Local Worker
+
+> Stability: 1 - Experimental
+
+### Class: `NodeRealm`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Extends: {EventEmitter}
+
+A `NodeRealm` is effectively a Node.js environment that runs within the
+same thread.
+
+```mjs
+import { NodeRealm } from 'vm';
+import { fileURLToPath } from 'url';
+const w = new NodeRealm();
+const myAsyncFunction = w.createRequire(fileURLToPath(import.meta.url))('my-module');
```

Comment:
nit: we may want to use a variable name other than `w` since this is no longer a worker.

Reply:
Personally I hate single-character variable names. In code, and _especially_ in documentation.

Reply:
```suggestion
const noderealm = new NodeRealm();
const myAsyncFunction = noderealm.createImport(import.meta.url)('my-module');
```